.buildkite/scripts/build_packages.sh

```diff
@@ -99,6 +99,11 @@ use_elastic_package
 
 echo "--- Build packages"
 
+if [[ "$BUILDKITE_RETRY_COUNT" != "0" ]]; then
+    echo "Please, trigger a new build to avoid issues publishing packages duplicating the artifacts in this build. Please trigger a new build"
```

Comment:
Nit. Redundant message?
```suggestion
    echo "Please, trigger a new build to avoid issues publishing packages duplicating the artifacts in this build."
```

Reply:
Thanks! Yes, it was a leftover when I rephrased the message.